lib/config/config-initializer.js

```diff
@@ -354,14 +353,9 @@ function promptUser(callback) {
             if (earlyAnswers.styleguide === "airbnb" && !earlyAnswers.airbnbReact) {
                 earlyAnswers.styleguide = "airbnb-base";
             }
-            try {
-                config = getConfigForStyleGuide(earlyAnswers.styleguide);
-                writeFile(config, earlyAnswers.format);
-            } catch (err) {
-                callback(err);
-                return;
-            }
-            return;
+
+            config = getConfigForStyleGuide(earlyAnswers.styleguide);
+            writeFile(config, earlyAnswers.format);
         }
 
         // continue with the questions otherwise...
```

Comment:
I think you're missing a `return` statement on the `inquirer.prompt` call on the line below this one. If an error occurs in the second or third `inquirer.prompt` call, it won't bubble back to the handler.